plugins/faustwp/includes/replacement/callbacks.php

```diff
@@ -95,40 +102,43 @@ function image_source_replacement( $content ) {
  *
  * @param array $sources One or more arrays of source data to include in the 'srcset'.
  *
- * @return string One or more arrays of source data.
+ * @return array One or more arrays of source data.
  */
 function image_source_srcset_replacement( $sources ) {
```

Comment:
I think we need to improve the function a little bit. It is definitely worth to keep initial comments here. And I guess there is a chance for us to make it more readable along with more compact.

Reply:
Ah that was updated when I ran the [phpcbf fix ](https://github.com/wpengine/faustjs/blob/canary/plugins/faustwp/composer.json#L45)

I will look at seeing what we can improve in terms of readability..

Reply:
Updated PHPDocs 👍